components/brave_news/browser/resources/context.tsx

```diff
@@ -9,8 +9,11 @@ import {
 import * as React from 'react'
 import usePromise from '../../../brave_new_tab_ui/hooks/usePromise'
 
+export type FeedType = 'all' | `publishers/${string}` | `channels/${string}`
```

Comment:
This is going to conflict with https://github.com/brave/brave-core/pull/20452

Reply:
It's interesting that we can use template literal like this 👍

Reply:
https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html

They're really handy, especially when you have combinations of strings that'd be painful to write out manually:

```
type Vertical = 'top' | 'bottom' | 'center'
type Horizontal = 'left' | 'right' | 'center'
type Align = `${Vertical}-${Horizontal}` | `center`